src/components/sidebar/sideBar.css

```diff
@@ -0,0 +1,50 @@
+.sidebar {
+  position: fixed;
+  top: 0;
+  right: 0;
+  right: -500px;
+  width: 500px;
+  height: 100%;
+  text-align: center;
+  background-color: #fff;
+  box-shadow: -5px 0 15px rgba(0, 0, 0, 0.5);
```

Comment:
Use one color system for whole app to be consistent

Reply:
Thank's!!!